sinch/core/deserializers.py

```diff
@@ -0,0 +1,5 @@
+from datetime import datetime
+
+
+def timestamp_to_datetime_in_utc_deserializer(timestamp: str):
+    return datetime.fromisoformat(timestamp + 'Z')
```

Comment:
In case the server changes their implementation, it would be good to check if the timezone is present or not

Reply:
I think we should release a new version of SDK when the API changes.

Reply:
btw. I don't think the API will change. As you know we have customers that have codebases that use this timestamp format. IMO we should work on healthy API versioning strategy first and foremost.

Reply:
TBH, as we are not part of the API governance, I don't think it's our call to make conjonctures about how the backend will evolve, if they will update the value or not, if they will make a new version or not...
The only thing we can ensure is that the code works no matter whether it receives a date with a timezone or not. And for robustness across all APIs, I would recommend to use this deserializer for all dates identified in the specification

Reply:
> I think we should release a new version of SDK when the API changes.

What will happen in the meanwhile?

Reply:
This discussion is way beyond this PR :) We can discuss this on architecture meeting.
I'll keep my implementation for now.